recipes/xtensor-python/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "xtensor-python" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "4306dbcc53e697cdf33abf9fbdd033c59a15510d359ed3998821a0085f66bbdd" %} 
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/QuantStack/xtensor-python/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - xtensor ==0.1.1
+    - pybind11 ==1.8.1
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/xtensor-python  # [unix]
+    - test -f ${PREFIX}/include/xtensor-python/pyarray.hpp  # [unix]
+    - if exist %PREFIX%\include\xtensor-python\pyarray.hpp (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://quantstack.net/xtensor
+  license: BSD
+  license_file: LICENSE
+  summary: 'Python bindings for xtensor, the C++ tensor algebra library'
+  description: 'Multi dimensional arrays with broadcasting and lazy computing'
+  doc_url: http://xtensor.readthedocs.io
+  dev_url: https://github.com/QuantStack/xtensor-python
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - JohanMabille
```

Comment:
Please confirm that it is ok for you to be a maintainer on this, @JohanMabille .